apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestAudioActivity.java

```diff
@@ -437,6 +443,24 @@ public void onClick(View v) {
         }
         OboeAudioStream.setCallbackReturnStop(false);
 
+        mHangTimeSpinner = (Spinner) findViewById(R.id.spinner_hang_time);
```

Comment:
What happens for activities for that open both input and output streams? Do both get hung by the same amount